docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +262,37 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+You can view metrics in Dynatrace dashboards:
+
+- To view the status of the Dynatrace integration with the Kyma Telemetry module, import the file [Telemetry Module Status](./telemetry-resource-metrics.json) as a Dynatrace dashboard. For details on how to import a dashboard, see [Importing Dashboards](https://docs.dynatrace.com/docs/analyze-explore-automate/dashboards-classic/dashboards/dashboard-json#import-dashboard).
+> [!WARNING]
```

Comment:
Does this have to happen before importing the file as a Dynatrace dashboard, or afterwards? (or doesn't it matter?)

Furthermore, does this have to be a warning? Why not simply another instruction:

```suggestion
2. Add the following attributes to the allow list of OpenTelemetry metrics resource attributes:
   - `k8s.resource.name`
   - `k8s.resource.group`
   - `k8s.resource.kind`
   - `k8s.resource.version`

   For details about adding attributes to the allow list, see [Configure resource and scope attributes to be added as dimensions](https://docs.dynatrace.com/docs/ingest-from/opentelemetry/getting-started/metrics/configuration#allow-list).
```

If this suggestion is acceptable, you can drop lines 272-278.